src/App.tsx

```diff
@@ -15,29 +15,16 @@ import { actions as AddTodosAction } from './features/todos';
 
 export const App: React.FC = () => {
   const [isLoading, setIsLoading] = useState(true);
-  const { query, status } = useAppSelector(state => state.filter);
   const currentTodo = useAppSelector(state => state.currentTodo);
-  const todos = useAppSelector(state => state.todos);
   const dispatch = useAppDispatch();
 
-  const filteredTodos = todos.filter(todo => todo.title.toLowerCase().includes(query.toLowerCase()))
-    .filter(todo => {
-      switch (status) {
-        case 'active':
-          return !todo.completed;
-
-        case 'completed':
-          return todo.completed;
-
-        default:
-          return todo;
-      }
-    });
-
   useEffect(() => {
-    getTodos()
-      .then((todosFromServer: Todo[]) => dispatch(AddTodosAction.addTodos(todosFromServer)))
-      .finally(() => setIsLoading(false));
+    (async () => {
+      const todosFromServer: Todo[] = await getTodos();
+
+      dispatch(AddTodosAction.addTodos(todosFromServer));
+      setIsLoading(false);
+    })();
```

Comment:
Move function declaration from useEffect to a separate function and just call it in useEffect()
```
const getTodosFromServer = useCallback( async () ...

useEffect(() => {
 getTodosFromServer()
}, [getTodosFromServe])